### PR TITLE
[Activation] Merge to Activation branch - 22082018

### PR DESCRIPTION
**Pull request by bokobza**
58 tests failing
image

Can @rowandh @codingupastorm @zeptin @thefazzer and @monsieurleberre give this a look, see if I haven't forgotten anything?

### CODE REVIEW

File: src/Stratis.SmartContracts.Core/BloomData.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Linq;
+using NBitcoin;
+
+namespace Stratis.SmartContracts.Core
+{
+    /// <summary>
+    /// Type representation of data used in a bloom filter.
+    /// </summary>
+    public class BloomData : IBitcoinSerializable
```
Comment (codingupastorm) [inline src/Stratis.SmartContracts.Core/BloomData.cs]: This file should be removed

File: src/Stratis.Bitcoin.Features.SmartContracts.Tests/BloomDataTests.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Stratis.SmartContracts.Core;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.SmartContracts.Tests
```
Comment (codingupastorm) [inline src/Stratis.Bitcoin.Features.SmartContracts.Tests/BloomDataTests.cs]: This file should be removed

**Comment by bokobza** [comment]
@codingupastorm here you go

**Comment by codingupastorm** [comment]
Thanks all good from me now @bokobza

**Comment by zeptin** [comment]
Looks good to me too

**Comment by thefazzer** [comment]
@bokobza Approved.

**Comment by bokobza** [comment]
Ok let's merge

**Pull request merged**